composer.json

```diff
@@ -26,7 +26,7 @@
     "php": ">=7.2",
     "ext-json": "*",
     "contributte/psr7-http-message": "~0.7.0",
-    "doctrine/annotations": "^1.8.0",
+    "doctrine/annotations": "^1.13",
```

Comment:
- Since 1.12 the `Doctrine\Common\Annotations\Annotation\NamedArgumentConstructor` class is present, but that's not enough.
- Since 1.13 the sole agument counts as default (with no array) = it finally behaves like attributes since this version



For more, I wrote a post about this process → https://tomasvotruba.com/blog/doctrine-annotations-and-attributes-living-together-in-peace/